packages/oasis-actions/src/strategies/aave/close.ts

```diff
@@ -64,24 +186,36 @@ async function getSwapData(args: AAVECloseArgs, dependencies: AAVECloseDependenc
     collateralTokenAddress,
     debtTokenAddress,
     swapAmountAfterFees,
-    args.slippage,
+    slippage,
   )
 
   return { swapData, collectFeeFrom, preSwapFee }
 }
 
-function calculatePreSwapFeeAmount(
-  collectFeeFrom: 'sourceToken' | 'targetToken' | undefined,
-  swapAmountBeforeFees: BigNumber,
+function calculateNeededCollateralToPaybackDebt(
+  debtPrice: BigNumber,
+  debtPrecision: number,
+  colPrice: BigNumber,
+  colPrecision: number,
+  debtAmount: BigNumber,
+  fee: BigNumber,
+  slippage: BigNumber,
 ) {
-  return collectFeeFrom === 'sourceToken'
-    ? calculateFee(swapAmountBeforeFees, new BigNumber(DEFAULT_FEE), new BigNumber(FEE_BASE))
-    : ZERO
+  const paybackAmount = debtPrice.times(debtAmount)
```

Comment:
```suggestion
    // Debt per protocol base currency x DebtAmount = Payback Amount (Units in base currency)
  const paybackAmount = debtPrice.times(debtAmount)
```

Reply:
If you think worth it. Just wrote comment in response to existing comment that is should be double checked